.jenkins

```diff
@@ -138,7 +138,7 @@ pipeline {
                               cmake \
                                 -DCMAKE_BUILD_TYPE=Debug \
                                 -DCMAKE_CXX_COMPILER=clang++ \
-                                -DCMAKE_CXX_FLAGS="--gcc-toolchain=$GCC_DIR -Werror" \
+                                -DCMAKE_CXX_FLAGS="-Wno-unknown-cuda-version -Werror" \
```

Comment:
Without this flag we get warnings about using a CUDA version newer than 10.1

Reply:
Yes, this is also needed for SYCL+CUDA.

Reply:
It's nice to not have warnings but I think this is not harming anything. Or am I wrong?